include/swift/SIL/ApplySite.h

```diff
@@ -90,28 +90,30 @@ class ApplySite {
 
   SILModule &getModule() const { return Inst->getModule(); }
 
-  static ApplySite isa(SILNode *node) {
-    auto *i = dyn_cast<SILInstruction>(node);
-    if (!i)
-      return ApplySite();
-
-    auto kind = ApplySiteKind::fromNodeKind(i->getKind());
+  static ApplySite isa(SILInstruction *inst) {
```

Comment:
@eeckstein I wish you had also renamed this to cast or get. It is weird that this is isa since it doesn't return a bool.